### PR TITLE
[Docs] update Japanese translation of feature_backlight.md

### PR DESCRIPTION
**Pull request by umi-umi**
## Description

update Japanese translation of docs/feature_backlight.md.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by mtei** [comment]
@shelaf @s-show @hsgw @satt99 @takashiski @yoshihara よかったらレビューお願いします。

File: docs/ja/feature_backlight.md [hunk 1]
```diff
@@ -67,14 +67,17 @@ BACKLIGHT_DRIVER = software
...
+| 定義                          | デフォルト         | 説明                                                                                          |
+| ----------------------------- | ------------------ | --------------------------------------------------------------------------------------------- |
+| `BACKLIGHT_PIN`               | *定義なし*         | LED を制御するピン                                                                            |
+| `BACKLIGHT_LEVELS`            | `3`                | 輝度のレベルの数 (オフを除いて最大 31)                                                        |
+| `BACKLIGHT_CAPS_LOCK`         | *定義なし*         | バックライトを使って Caps Lock のインジケータを有効にする (専用 LED の無いキーボードのため)   |
+| `BACKLIGHT_BREATHING`         | *定義なし*         | サポートされる場合は、バックライトの明滅動作を有効にする                                      |
+| `BREATHING_PERIOD`            | `6`                | 各バックライトの "明滅" の長さ（秒）                                                          |
+| `BACKLIGHT_ON_STATE`          | `1`                | バックライトが "オン" の時のバックライトピンの状態 - high の場合は `1`、low の場合は `0`      |
+| `BACKLIGHT_LIMIT_VAL`         | `255`              | バックライトの最大デューティサイクル -- `255` で完全な輝度になり、それ未満では最大値が減少する|
```
Comment (shelaf) [inline docs/ja/feature_backlight.md:78]:
```suggestion
| `BACKLIGHT_LIMIT_VAL`         | `255`              | バックライトの最大デューティサイクル -- `255` で最大輝度になり、それ未満では最大値が減少する|
```

**Review by shelaf — APPROVED** [review]

**Review by s-show — APPROVED** [review]

**Review by mtei — APPROVED** [review]

**Comment by mtei** [comment]
I think this is ready to merge.

**Comment by mtei** [comment]
Thanks!

**Pull request merged**